packages/caravan-bitcoin/src/types/addresses.ts

```diff
@@ -2,4 +2,4 @@
 // address type.
 // We should be able to replace this with use of the MULTISIG_ADDRESS_TYPES
 // enum when that file (./multisig.js) gets converted to typescript
-export type MultisigAddressType = "P2SH" | "P2WSH" | "P2SH-P2WSH" | "P2TR";
+export type MultisigAddressType = "P2SH" | "P2WSH" | "P2SH-P2WSH" | "P2TR" | "P2PKH" | "UNKNOWN";
```

Comment:
We can avoid UNKNOWN and instead show the error as it is if it occurs. It was just to keep the code consistant by adding a new default type in case of address type matching failure.

Reply:
Yeah, I think it should just fail if it doesn't match.